sri/utils.py

```diff
@@ -15,7 +16,7 @@ def get_static_path(path: str) -> Path:
     Resolves a path commonly passed to `{% static %}` into a filesystem path
     """
 
-    if hasattr(staticfiles_storage, "stored_name"):
+    if (not settings.DEBUG) and hasattr(staticfiles_storage, "stored_name"):
```

Comment:
Question: What about `DEBUG` is it that changes the behaviour here? Is there a better setting it should be derived from?